### PR TITLE
Add 5.17.1 release notes

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds release notes for 5.17.1 patch release

## Motivation and Context
Support 5.17.1 release

### CODE REVIEW

**Review by palourde — APPROVED** [review]
👌

File: content/sensu-go/5.17/release-notes.md [hunk 1]
```diff
@@ -51,6 +52,25 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+**IMPROVEMENTS:**
+
+- Asset names may now include capital letters.
+- Running the `sensuctl configure` command now resets the sensuctl cluster configuration.
+- When you use `--trusted-ca-file` to configure sensuctl, it now detects and saves the absolute file path in the cluster configuration.
+
+**FIXES:**
+
+- ([Commercial feature][106]) In the [web UI][107], polling is re-enabled for the silences view.
+- Fixed a bug that prevented `.tar` assets that contain hardlinked files from extracting.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
- Fixed a bug that prevented `.tar` assets from extracting if they contain hardlinked files.
```

File: content/sensu-go/5.17/release-notes.md [hunk 2]
```diff
@@ -51,6 +52,25 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.17.1 release notes
+
+**January 31, 2020** &mdash; The latest release of Sensu Go, version 5.17.1, is now available for download.
+This release fixes a web UI issue that cleared selected filters when sorting an event list and a bug that prevented certain `.tar` assets from extracting and. It also includes sensuctl configuration improvements.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
This release fixes a web UI issue that cleared selected filters when sorting an event list and a bug that prevented certain `.tar` assets from extracting.
It also includes sensuctl configuration improvements.
```

File: content/sensu-go/5.17/release-notes.md [hunk 3]
```diff
@@ -51,6 +52,26 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**IMPROVEMENTS:**
+
+- Asset names may now include capital letters.
+- Running the `sensuctl configure` command now resets the sensuctl cluster configuration.
+- When you use `--trusted-ca-file` to configure sensuctl, it now detects and saves the absolute file path in the cluster configuration.
+
+**FIXES:**
+
+- ([Commercial feature][106]) In the [web UI][107], polling is re-enabled for the silences view.
```
Comment (cwjohnston) [inline content/sensu-go/5.17/release-notes.md]: I believe this change should be described differently. As I understand it from https://github.com/sensu/sensu-enterprise-go/issues/853, users may have noticed silencing entries lingering in the web UI after they had been removed. The fix ensures that the web UI is updated when a silencing entry is removed or expires.

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.17/release-notes.md (hunk 3 above)
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
- ([Commercial feature][106]) When a silencing entry expires or is removed, it is also removed from the silences view in the [web UI][107].
```

**Pull request merged**